docs/eventing/sink-binding.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Sink Binding"
+weight: 60
+type: "docs"
+---
+
+![version](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` custom object supports decoupling event production from delivery addressing.
+
+You can use sink binding to connect event producers, such as an event source, to an addressable Kubernetes object that can receive events, also known as an event sink.
+
+Sink binding injects environment variables into the `PodTemplateSpec` of the event sink, so that the application code does not need to interact directly with the Kubernetes API to locate the event destination.
+
+# Prerequisites
+
+Before you can use sink binding, you must ensure that:
+- Knative Serving and Eventing are installed on your cluster.
+- Optional: If you want to use `kn` commands with sink binding, you must install the `kn` CLI.
+
+# Procedure
+
+1. Create a Knative service if you do not have an existing event sink that you want to use.
+  ###### Example Knative `Service` object:
+  {{< tabs name="knative_service" default="kn" >}}
+  {{% tab name="kn" %}}
+  ```
+  kn service create hello --image gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display --env RESPONSE="Hello Serverless!"
+  ```
+  {{< /tab >}}
+  {{% tab name="yaml %}}
+  ```yaml
+  apiVersion: serving.knative.dev/v1
+  kind: Service
+  metadata:
+    name: event-display
+  spec:
+    template:
+      spec:
+        containers:
+          - image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+  ```
+  {{< /tab >}}
+  {{< /tabs >}}
+1. Create a cron job if you do not have an existing event source that you want to use.
+  ###### Example `CronJob` object:
+
+  ```yaml
+  apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: heartbeat-cron
+  spec:
+    # Run every minute
+    schedule: "* * * * *"
+    jobTemplate:
+      metadata:
+        labels:
+          app: heartbeat-cron
+      spec:
+        template:
+          spec:
+            restartPolicy: Never
+            containers:
+              - name: single-heartbeat
+                image: <FILL IN YOUR IMAGE HERE>
```

Comment:
We release this test image, @mattmoor there is a macro for this?

Reply:
not afaik

Reply:
@n3wscott could you drop me some accurate sample YAML if you have it handy with the right image etc? :slightly_smiling_face: thank you

Reply:
I think we are looking at making a central place for these, I don't have a solid release at the moment.

Reply:
@n3wscott any update here or is this OK to go without adding for now?